tmt/trying.py

```diff
@@ -388,6 +399,29 @@ def action_quit(self, plan: Plan) -> None:
         run_id = click.style(plan.my_run.workdir, fg="magenta")
         self.print(f"Run {run_id} successfully finished. Bye for now!")
 
+    def handle_options(self, plan: Plan) -> None:
```

Comment:
This seems like it's designed to only handle one option, or am I missing something?  
I'd expect `option` list to be `options` and then instead of `option[0]` have a loop for all valid options in the list.

Reply:
Indeed, the method does handle just a single option - adding new options to `self.cli_options` and to command would still require this method to be expanded, otherwise only the first one met in `option` below would be handled.

Reply:
Hm, can we keep it as it is for now and when I will start with the other option, i will expand the handle_options usage?